src/Http/Session.php

```diff
@@ -447,14 +448,30 @@ public function read(?string $name = null)
             return $_SESSION ?: [];
         }
 
-        return Hash::get($_SESSION, $name);
+        return Hash::get($_SESSION, $name, $default);
+    }
+
+    /**
+     * Returns given session variable, or throws Exception if not found.
+     *
+     * @param string $name The name of the session variable (or a path as sent to Hash.extract)
+     * @throws \RuntimeException
+     * @return mixed|null
+     */
+    public function readOrFail(string $name)
+    {
+        if ($this->check($name) === false) {
```

Comment:
```suggestion
        if (!$this->check($name)) {
```

Reply:
Sure, thats fair. I just copied this from the other check() method, and as such it should be the same in both cases, though.